tests/integrate_test/test_block_interval.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+
+import argparse
+import time
+from txtool_utils import get_block_by_number, get_current_block_number, send_tx, get_receipt, get_metadata
+
+
+def main():
+    print(f'a) Test default block interval...')
+    current_height = get_current_block_number()
+    height = int(current_height, 16)
+
+    time.sleep(3)
+
+    next_height = str(hex(height + 1))
+    next_block = get_block_by_number(next_height)
+    assert next_block != None, "Default block interval is 3s"
+    metadata = get_metadata()
+    assert metadata['blockInterval'] == 3000
+
+    print(f'b) Send transaction to change block interval...')
+    # send tx to change the block interval from 3s to 6s
+    admin_privkey = '0x5f0258a4778057a8a7d97809bd209055b2fbafa654ce7d31ec7191066b9225e6'
+    system_contract_address = '0xffffffffffffffffffffffffffffffffff020000'
+    code = 'de7aa05d0000000000000000000000000000000000000000000000000000000000001770'  # setBlockInterval(6000)
+
+    opts = parse_arguments()
+    version = opts.version
+
+    tx_hash = send_tx(
+        admin_privkey, to=system_contract_address, code=code, version=version)
+    time.sleep(6)
+    receipt = get_receipt(tx_hash)
+    assert receipt['errorMessage'] == None, "Transaction execution failed"
+
+    print(f'c) Test changed block interval...')
+    current_height = get_current_block_number()
+    height = int(current_height, 16)
+    next_height = str(hex(height + 1))
+
+    time.sleep(3)
```

Comment:
IMHO, This is a very unstable test.

Reply:
A little..., may be just check  metadata is ok.

Reply:
You can try to set the `block interval` value to a big number. then use the `check_height_stopped` with a resonable timeout.

Reply:
Please check again. I do some modifications to make this test stable.

Reply:
But `sleep` is the real reason of unstable not just `3s`.